ScreenBoom/ScreenBoom/View/RightMenuView.swift

```diff
@@ -11,39 +11,40 @@ import UIKit
 
 class RightMenuView: UIView {
   
+  let eventViewModel = EventViewModel()
+  let imageNames = ImageNames()
+  var sideMenuDelegate : SideMenuDelegate?
+  
   let blurEffectView : UIVisualEffectView = {
     let view = UIVisualEffectView()
     view.effect = UIBlurEffect(style: UIBlurEffectStyle.dark)
     return view
   }()
   let editButton : UIButton = {
     let view = UIButton()
-    view.isHidden = true
+//    view.isHidden = true
     view.setTitle("Edit", for: .normal)
     view.titleLabel?.textColor = UIColor.blue
     view.titleLabel?.backgroundColor = UIColor.clear
     view.backgroundColor = UIColor.clear
     return view
   }()
-  let pauseImage : UIImageView = {
-    let view = UIImageView()
-    view.isHidden = true
-    view.image = UIImage(named: "pauseEnable")
-    view.isUserInteractionEnabled = true
+  let pauseButton : UIButton = {
+    let view = UIButton()
+//    view.isHidden = true
+    view.setImage(UIImage(named: "pauseEnable"), for: .normal)
     return view
   }()
-  let playImage : UIImageView = {
-    let view = UIImageView()
-    view.isHidden = true
-    view.image = UIImage(named: "playNotEnable")
-    view.isUserInteractionEnabled = false
+  let playButton : UIButton = {
+    let view = UIButton()
+//    view.isHidden = true
+    view.setImage(UIImage(named: "playNotEnable"), for: .normal)
     return view
   }()
-  let deleteImage : UIImageView = {
-    let view = UIImageView()
-    view.isHidden = true
-    view.image = UIImage(named: "delete")
-    view.isUserInteractionEnabled = true
+  let deleteButton : UIButton = {
+    let view = UIButton()
+//    view.isHidden = true
+    view.setImage(UIImage(named: "delete"), for: .normal)
```

Comment:
a lot of these buttons are really similar. Consider creating a subclass of UIButton called SideMenuButton for example that will take and image and a title so that we have a clear component that we can add stuff to later.